packages/aws-lambda/src/wrapper.js

```diff
@@ -401,16 +401,32 @@ function postHandler(entrySpan, error, result, callback) {
  * back end.
  */
 function postHandlerForTimeout(entrySpan, remainingMillis) {
```

Comment:
Even if we would make this optional for the customer the error count can occur still if the extension does not receive a request from node. That's why we need the go fix too.

It's still an option to make this optional.

Reply:
Yes, let's add an opt-out for the heuristical timeout detection 👍